codechecker_lib/session_manager.py

```diff
@@ -305,29 +310,22 @@ def __init__(self):
         LOG.debug('Loading session config')
 
         # Check whether user's configuration exists.
-        session_cfg_file = os.path.join(os.path.expanduser("~"),
+        user_home = os.path.expanduser("~")
+        session_cfg_file = os.path.join(user_home,
                                         ".codechecker_passwords.json")
-        if not os.path.exists(session_cfg_file):
-            LOG.info("CodeChecker authentication client's example "
-                     "configuration file created at " + session_cfg_file)
-            shutil.copyfile(os.path.join(os.environ['CC_PACKAGE_ROOT'],
-                                         "config", "session_client.json"),
-                            session_cfg_file)
-            os.chmod(session_cfg_file, stat.S_IRUSR | stat.S_IWUSR)
```

Comment:
Is it okay to not set the permissions so this file is only readable by the owner? Is that the default permission on every system we support?